client/packages/android/app/src/main/assets/capacitor.config.json

```diff
@@ -1,6 +1,6 @@
 {
 	"appId": "org.openmsupply.client",
-	"appName": "openmsupply-client",
+	"appName": "Open mSupply",
```

Comment:
does that just rename how the app name looks, it doesn't effect anything else (like the upgrade etc ?)

Reply:
the upgrade worked fine - upgrading works off the id, so when it upgraded it just changed the display name